packages/ui/react/src/components/Primitives/Dropdown/Dropdown.test.tsx

```diff
@@ -0,0 +1,85 @@
+import '@testing-library/jest-dom'
+
+import { act, render, screen } from '@testing-library/react'
+import { describe, expect, it, vi } from 'vitest'
+
+import { Dropdown } from '.'
+
+describe('Dropdown', () => {
+  it('render correctly with required props', () => {
+    render(
+      <Dropdown
+        id="test-dropdown"
+        anchorType="button"
+        label="Menu"
+        options={[]}
+      />
+    )
+
+    const subjectDropdown = screen.getByText('Menu')
+    expect(subjectDropdown).toBeInTheDocument()
+  })
+
+  it('show dropdown options when anchor is clicked', () => {
+    render(
+      <Dropdown
+        id="test-dropdown"
+        anchorType="button"
```

Comment:
@thlmenezes could the test warning be fixed as part of this PR?
https://github.com/devopness/devopness/actions/runs/11731739335/job/32682418466?pr=1007#step:7:17


<img width="1356" alt="image" src="https://github.com/user-attachments/assets/bb7ef69e-f0a9-4076-b0cd-9befa414cc8c">

Reply:
`noIconMargin` and `noMargin`: Will be fixed in #1004 

`anchorType`: fixed

https://github.com/devopness/devopness/actions/runs/11743481139/job/32716424711?pr=1007

![image](https://github.com/user-attachments/assets/186200ef-9d6b-49be-9846-934d5406a29b)